src/main/java/io/appium/java_client/android/AndroidDriver.java

```diff
@@ -176,4 +179,16 @@ public void openNotifications() {
     public void toggleLocationServices() {
         CommandExecutionHelper.execute(this, toggleLocationServicesCommand());
     }
+
+    public void toggleWifi() {
+        CommandExecutionHelper.execute(this, toggleWifiCommand());
+    }
+
+    public void toggleAirplaneMode() {
+        CommandExecutionHelper.execute(this, toggleAirplaneCommand());
+    }
+
+    public void toggleData() {
```

Comment:
This only works on Android. Also, the device should be rooted

Reply:
Shouldn't the server thrown exception in this case ?